sherpa/optmethods/tests/test_optmethods.py

```diff
@@ -115,6 +117,20 @@ def test_gaussian(self):
         self.tst( optfcts.lmdif, name + self.lm, _tstoptfct.gaussian,
                   fmin, x0, xmin, xmax )
 
+    def test_mccormick(self):
```

Comment:
@dtnguyen2 - `test_mccormick` passes on master branch (ie without your change above to use `reflect_about_boundary`). Is this expected?